### DISCUSSION TITLE
Cameras not refreshing

### DISCUSSION THREAD

**Discussion by dwdonnelly** [mrlt8/docker-wyze-bridge#1282] [Q&A] [answered]
Just installed the Docker Wyze Bridge.  Install produced no errors as viewed in the log.  In the configuration tab, set wyse e-mail, pwd, Key ID, and API Key, selected ANY under Allowed Net Modes, set Snapshot Mode for RTSP60, selected "SAVE".  (note: when I return to this configuration page, none of my configuration shows - is this correct?)

Open WEB UI, all cameras appear, meaning:  I have multiple cameras disconnected, but they show just blank screens (that is expected).  The active cameras are two interior and three external.  The two internals show and update as expected on the WEB UI,  1 of the external cameras has no image, and will not update, I just get a constant revolving circle, waiting to update.  The other two external cameras show an image, but will not update.  I have verified via the Wyze app, that all active cameras are working and display correctly in the Wyze App.

Your help is appreciated.

**Comment by mrlt8** [comment]
You can set `IGNORE_OFFLINE` to true to ignore all the offline cameras, or use `FILTER_NAMES` to either filter in or filter out the cameras as needed.

As for camera that is connected and not working, is it by any chance a battery-powered one? The snapshot will NOT update by default to prevent draining the battery, but they should update if the stream is active.

**Comment by dwdonnelly** [comment]
@mrlt8 Went in and selected `IGNORE_OFFLINE` cameras.  All but one of the offline cameras are now hidden (not showing).  The one offline camera that is showing, is an external (not connected/not in use) camera - and yes - it is battery powered.

I then further ran a test based on your "battery" comment.  I went outside and walked in front of the two external battery-powered cameras.  Came back in and sure enough - the one camera's image changed (to what I expected) - it showed me looking at the camera.  The other external battery-powered camera's image did not change - but that is based on my tweaking the detection area/sensitivity in the Wyze App.  I am confident it will report as the other one did once I get it tweaked in.

So, I am guessing that for the one external battery-powered camera that is still showing, I will need to `FILTER_NAMES`?  And if that is the case - having combed through the configuration tab, I only see a filter by name in the MQTT area - which, again, I am guessing is not the right field.

I read this: https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Filtering, but not sure how to use.  Meaning, not sure how to get to the `docker-compose.yml`.  Is that via the WEB UI and select `edit in yaml`?  Would this adding the filter by name override the ignore offline camera setting or simply add to it?

**Reply by mrlt8** [reply, marked as answer]
You should be able to filter in/out the cameras via the add-on config GUI by toggling "Show unused optional configuration options":

<img width="1048" alt="homeassistant" src="https://github.com/mrlt8/docker-wyze-bridge/assets/67088095/0de8ee99-2ce6-438d-bdea-a2c452fa18c5">

**Comment by dwdonnelly** [comment]
@mrlt8 Yup, that fixed it.  I saw that field but presumed it was part of the MQTT field set as it was grouped within.  Many thanks !  Again - great work - it is much appreciated!
